### DISCUSSION TITLE
Web apps to follow Container setup

### DISCUSSION THREAD

**Discussion by Zorakie** [Floorp-Projects/Floorp#1247] [Feature Request]
If I set up a page as a web app, it should have a setting for it to be opened in a set Container.

Usecase: For my work environment, I have my company's O365 environment logged in in my default tab/session (= no container). That session is also used to use O365 SSO across other cloud platforms that I use for work.
But then I work on a customer's project and the customer provides us with their own O365 account and their SSO for their platforms that I need to work on for that project.

My current problem is, that I need to install a customer's MS TEAMS as a web app. But when I do so, even though I install it from containered tab, when I open the web app, it has no container, so it opens my company teams, instead of the containerized customer teams.

Could that be added, please?

**Comment by SystemError12** [comment]
Check this topic: 

https://github.com/Floorp-Projects/Floorp/discussions/1061
